py3status/modules/whatismyip.py

```diff
@@ -49,25 +57,33 @@ def on_click(self, i3s_output_list, i3s_config, event):
     def _get_my_ip(self):
         """
         """
+        country = None
```

Comment:
In other modules we have used `?` when things are unknown eg battery_level.

`None` will get shown as `None` so if you just want the county to be blank use `''`